### PR TITLE
feat(console,core)!: use rbac scope to control management resource

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

use RBAC's scope to control management api and admin console access.

## Testing

Pass all tests.

Local tested with admin user and guest user, as expected.

### CODE REVIEW

File: packages/core/src/routes/interaction/consent.ts [hunk 1]
```diff
@@ -31,10 +35,13 @@ export default function consentRoutes<T>(
 
     // Temp solution before migrating to RBAC. Block non-admin user from consenting to admin console
```
Comment (xiaoyijun) [inline packages/core/src/routes/interaction/consent.ts:36]: We need to delete this comment.

**Review by xiaoyijun — APPROVED** [review]
LGTM

File: packages/console/package.json [hunk 2]
```diff
@@ -23,7 +23,7 @@
     "@logto/language-kit": "workspace:*",
     "@logto/phrases": "workspace:*",
     "@logto/phrases-ui": "workspace:*",
-    "@logto/react": "1.0.0-beta.14",
+    "@logto/react": "1.0.0-beta.15",
```
Comment (charIeszhao) [inline packages/console/package.json:26]: We need to also bump the react sdk version in demo-app and integration-tests

**Review by charIeszhao — APPROVED** [review]

File: packages/console/package.json (hunk 2 above)
Comment (wangsijie) [reply packages/console/package.json:26]: updated

**Pull request merged**